build/yamls/base/conf/antrea-agent.conf

```diff
@@ -32,3 +32,10 @@
 # CIDR Range for services in cluster. It's required to support egress network policy, should
 # be set to the same value as the one specified by --service-cluster-ip-range for kube-apiserver.
 #serviceCIDR: 10.96.0.0/12
+
+# Determines how pod traffic is encapsulated. It has the following options
```

Comment:
We generally capitalize the first letter of K8s resource type, like Pod, Node, NetworkPolicy, etc..